src/app/scripts/cac/control/cac-control-sidebar-explore.js

```diff
@@ -215,6 +215,15 @@ CAC.Control.SidebarExplore = (function ($, BikeOptions, MapTemplates, Routing, T
      * @param {Object} $container jQuery-selected HTML snippet in the sidebar for the destination
      */
     function setDestinationDistance(destination, $container) {
+
+        // first check if we have distance cached
+        if (destination.durationMinutes) {
+            $container.find(options.selectors.distanceMinutesText)
+                .text(destination.durationMinutes + ' minutes away');
```

Comment:
Since this bit of code now appears twice, it may be worth abstracting it to a function that takes a $container and number of minutes. Then if we need to change the formatting, it can be done in a single place.